packages/rum-core/src/performance-monitoring/performance-monitoring.js

```diff
@@ -188,6 +188,14 @@ export default class PerformanceMonitoring {
         task.eventType === 'click'
       ) {
         const target = task.target
```

Comment:
Getting the follow stacktrace:
```
TypeError: invalid assignment to const 'target'
    getEventTargetSub performance-monitoring.js:197
    sendOnly event-handler.js:35
    sendOnly event-handler.js:33
    send event-handler.js:45
    patchAll index.js:23
    ...
```

Preferably `let target = task.target` here so [line 197](https://github.com/elastic/apm-agent-rum-js/pull/951/files#diff-e3d6e2c6dedbb088511627f63de9cfdfed996ea94849ccf78f155b171cb33a5eR197) can update the var?

Reply:
Fixed, and added the eslint rule.

https://eslint.org/docs/rules/no-const-assign